backend/zeno_backend/database/select.py

```diff
@@ -884,6 +887,25 @@ def project_state(project_uuid: str, project: Project) -> ProjectState | None:
         )
         models = [m[0] for m in model_results]
 
+        editor = db.execute_return(
+            "SELECT editor FROM user_project WHERE user_id = %s AND project_uuid = %s;",
+            [user.id, project_uuid],
+        )
+        if len(editor) == 0:
+            project.editor = False
+        else:
+            project.editor = bool(editor[0][0])
+
+        if not project.editor:
+            org_editor = db.execute_return(
+                "SELECT editor FROM organization_project AS op JOIN user_organization "
+                "AS uo ON op.organization_id = uo.organization_id "
+                "WHERE uo.user_id = %s AND op.project_uuid = %s AND uo.admin = TRUE;",
+                [user.id, project_uuid],
+            )
+            if len(org_editor) > 0:
+                project.editor = bool(org_editor[0][0])
```

Comment:
I don't think this is correct. The user does not have to be an admin of the organization to have access and it is sufficient if any of the orgs has edit rights on a project, not necessarily the first one.

Reply:
If we do it like this everywhere, I think it's wrong everywhere.

Reply:
For (A) this is just for editing, not for viewing check.

For (B) good catch again and I think it might be wrong in other places too.

Reply:
Even for A, one does not have to be an Org admin to edit if the org has edit rights, correct? Org admin is so you can edit the org settings, not the project settings.

Reply:
oh lord okay we should clean up the differences between admin and editor maybe, that makes sense.

Reply:
Yep, 100%